src/SonarScanner.MSBuild.Shim/PathHelper.cs

```diff
@@ -35,10 +35,6 @@ public static string WithTrailingDirectorySeparator(this DirectoryInfo directory
             {
                 return directory.FullName;
             }
-            else if (directory.FullName.Contains(Path.AltDirectorySeparatorChar))
```

Comment:
this scenario is covered by https://github.com/SonarSource/sonar-scanner-msbuild/blob/0ee3e6e2b38a6f1106fd5d057c4f2e81178927f9/Tests/SonarScanner.MSBuild.Shim.Test/PathHelperTests.cs#L64

However `FullName` won't ever contain the `AltDirectorySeparatorChar` , at least not on our current version of the framework.

Either way, this is only a Windows specific scenario and Windows allows mixed path separators, so better to remove code that we cannot test given it won't fail.